sql/updates/2.10.1--2.10.2.sql

```diff
@@ -11,8 +11,8 @@ DROP VIEW IF EXISTS timescaledb_experimental.policies;
 
 ALTER TABLE _timescaledb_config.bgw_job
     ALTER COLUMN owner DROP DEFAULT,
-    ALTER COLUMN owner TYPE regrole USING owner::regrole,
-    ALTER COLUMN owner SET DEFAULT current_role::regrole;
+    ALTER COLUMN owner TYPE regrole USING quote_ident(owner)::regrole,
+    ALTER COLUMN owner SET DEFAULT quote_ident(current_role)::regrole;
```

Comment:
You can't change existing update/downgrade SQL scripts, you should create a new update/downgrade scripts in latest-dev.sql and reverse-dev.sql because users that already updated from 2.10.1 to 2.10.2 will not have your change

Reply:
> You can't change existing update/downgrade SQL scripts, you should create a new update/downgrade scripts in latest-dev.sql and reverse-dev.sql because users that already updated from 2.10.1 to 2.10.2 will not have your change.

Not sure I follow the reasoning.

A user that already upgraded to 2.10.2 without error did not have a case-sensitive user name. If they had, they would fail to upgrade. In this case, the user name is the same before and after the upgrade.

A user that fails to upgrade to any version that includes this file (for example, 2.10.1 to 2.10.3) will fail because the user does not exist (I think we can assume that since it is not very likely that there multiple users with same user name but different case spelling). In this case we need to correct this file.

> This change should be part of the next update script, for example: 2.10.2--2.10.3.sql

We need to correct the default in the next upgrade script as well, yes. If a case-sensitive user is added while on 2.10.2, they need to be able to update to the new version and not correcting the default would break this.

Reply:
@fabriziomello only updated latest-dev and reverse-dev now

Reply:
This is about consistent catalog definition across versions. If we modify update scripts that have already been packaged and distributed we end up with users having the same timescaledb version but different catalog definition which is something we should avoid.